src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentColumn.jsx

```diff
@@ -56,13 +55,6 @@ export default function AppointmentColumn({
       {...props}
       role="cell"
     >
-      {icon && (
-        <i
-          aria-hidden="true"
-          className={classNames('fas', 'vads-u-margin-right--1', icon)}
-        />
-      )}
-
```

Comment:
Removing this section since the icon property is never used by the dependency, `AppointmentColumnLayout` and `RequestAppointmentLayout`